CONTRIBUTING.md

```diff
@@ -209,7 +216,13 @@ py.test -v plugins/sqlfluff-templater-dbt/test/
 
 ### Pre-Commit Config
 
-For development convenience we also provide a `.pre-commit-config.yaml` file to allow the user to install a selection of pre-commit hooks via `tox -e pre-commit -- install`. These hooks can help the user identify and fix potential linting/typing violations prior to committing their code and therefore reduce having to deal with these sort of issues during code review.
+For development convenience we also provide a `.pre-commit-config.yaml` file to allow the user to install a selection of pre-commit hooks by running (check the [requirements](#requirements) before running this):
```

Comment:
And here. Do think we’ve gone from too little guidance here to too much. But maybe you feel strong about leaving some of these in place since I k ow you say you wasted a good bit of time on this?

```suggestion
For development convenience we also provide a `.pre-commit-config.yaml` file to allow the user to install a selection of pre-commit hooks by running:
```

Reply:
Out of the cases discussed, this is maybe the most important to keep the pointer. Because this is so far away from the main tox stuff, I can easily imagine someone jumping here just for the sake of installing pre-commit tools and again if that someone already has tox installed, it's not obvious that it could be wrong even though `tox -e pre-commit -- install` seems to succeed.

Reply:
To make things worse, if you already ran this before you realized you need to change your tox installation, it's not trivial to realize that you also need to wipe out the `.tox/pre-commit` by hand and run `tox -e pre-commit -- install` again. Better safe than sorry, it's a pity that we can't enforce these programmatically.